src/Util/include/FMI3/fmi3_enums.h

```diff
@@ -57,35 +57,36 @@ typedef enum fmi3_fmu_kind_enu_t
 {
         fmi3_fmu_kind_unknown   = 1 << 0,
         fmi3_fmu_kind_me        = 1 << 1,
-        fmi3_fmu_kind_cs       = 1 << 2,
-        fmi3_fmu_kind_se       = 1 << 3
+        fmi3_fmu_kind_cs        = 1 << 2,
+        fmi3_fmu_kind_se        = 1 << 3
 } fmi3_fmu_kind_enu_t;
 
 /** \brief Convert a #fmi3_fmu_kind_enu_t constant into string  */
 FMILIB_EXPORT const char* fmi3_fmu_kind_to_string(fmi3_fmu_kind_enu_t kind);
 
 /**  \brief Variability property for variables */
 typedef enum fmi3_variability_enu_t {
-        fmi3_variability_enu_constant = 0,
-        fmi3_variability_enu_fixed  = 1,
-        fmi3_variability_enu_tunable  = 2,
-        fmi3_variability_enu_discrete  = 3,
+        fmi3_variability_enu_constant    = 0,
+        fmi3_variability_enu_fixed       = 1,
+        fmi3_variability_enu_tunable     = 2,
+        fmi3_variability_enu_discrete    = 3,
         fmi3_variability_enu_continuous  = 4,
-        fmi3_variability_enu_unknown  = 5
+        fmi3_variability_enu_unknown     = 5
```

Comment:
Sure we can keep it for now, but I don't think it adds clarity. If I see explicit values in enums I expect there to be some smart thing about it. Like setting something to -1, or setting the first value to =1 (and leaving rest, to reserve =0 for future). I mean, you're never even supposed to use the actual value of the enum, since you always access it via its variable name.